code/go/0chain.net/smartcontract/dbs/event/curator.go

```diff
@@ -0,0 +1,59 @@
+package event
+
+import (
+	"errors"
+	"fmt"
+
+	"gorm.io/gorm"
+)
+
+type Curator struct {
+	gorm.Model
+
+	// Foreign Key
+	AllocationID string `json:"allocation_id"`
```

Comment:
can you create a separate issue for the same

Reply:
Ok

Reply:
https://github.com/0chain/0chain/issues/952